viewcontribute.php

```diff
@@ -224,7 +224,10 @@
             $contribtitle = file_rewrite_pluginfile_urls($contrib->title, 'pluginfile.php', $context->id, 'mod_giportfolio',
                                                          'contribution', $contrib->id);
             echo '<strong>'.$contribtitle.'</strong></br>';
-            echo date('l jS F Y', $contrib->timemodified);
+            echo date('l jS F Y'.($giportfolio->timeofday ? ' h:i A' : ''), $contrib->timecreated);
+            if($contrib->timecreated !== $contrib->timemodified) {
+            	echo '<br/><i>Last modified on '.date('l jS F Y'.($giportfolio->timeofday ? ' \a\t h:i A' : ''), $contrib->timemodified).'</i>';
```

Comment:
Output text like "Last modified on" needs to be in a language string, so it can be translated.

Ideally the time should be using the standard formats in langconfig, but I can see I wasn't doing this before (and I can't remember why not, but there was probably a good reason), so this is fine.